ucm2/Toradex/apalis/eval.conf

```diff
@@ -0,0 +1,12 @@
+# Use case configuration for Toradex Apalis iMX8, Evaluation Board v1.2
+
+Syntax 4
+
+SectionUseCase."HiFi" {
+        File "/Toradex/apalis/eval-HiFi.conf"
+        Comment "Default"
+}
+
+BootSequence [
+        cset "name='Headphone Volume' 50%"
```

Comment:
Overall: If hardware supports volume control, it should be speciified e.g. using `PlaybackMixerElem`.